### ISSUE TITLE
Log messages for compute, storage and data resources.

### ISSUE THREAD

**Issue by Zarquan** [ivoa/Calycopis-broker#42]
Based on Message Templates
https://messagetemplates.org/

**Comment by Zarquan** [comment]
```
LogMessage:
  description: >
    A log message based on the Message Templates standard.
    https://messagetemplates.org/
  type: object
  properties:
    type:
      description: >
        The message type identifier.
        Typically a URL pointing to a human readable description of the message.
      type: string
    time:
      description: >
        The date and time of the message.
      type: string
      format: date-time
    template:
      description: >
        The message template.
      type: string
    values:
      description: >
        A map of name->value properties.
      xml:
        name: values
      $ref: '#/components/schemas/NameValueMap'
```

**Comment by Zarquan** [comment]
Fixed in #51

**Comment by Zarquan** [comment]
Done - Everything derived from `AbstractComponent` has a `messages` block.
